frontend/src/devicelist/__tests__/DeviceList.test.jsx

```diff
@@ -119,11 +120,11 @@ function verifyStaticElements() {
 
   const streamButton = document.getElementById("DeviceListStreamBtn");
   expect(streamButton).not.toBe(null);
-  expect(streamButton.textContent).toBe(" Stream");
+  expect(streamButton.textContent).toBe("Stream");
 
   const addDeviceButton = document.getElementById("DeviceListAddDevBtn");
   expect(addDeviceButton).not.toBe(null);
-  expect(addDeviceButton.textContent).toBe(" Add Device");
+  expect(addDeviceButton.textContent).toBe("Add Device");
```

Comment:
Shouldn't that make the tests fail?

Reply:
Why?

Reply:
The expected text changed but I didn't see a similar change for actual text. Does `toBe()` ignore whitespaces?